forwards_paper.tex

```diff
@@ -147,38 +151,37 @@ \section*{Introduction}
+is equivalent to having generated them during the simulation:
+it does not matter if we generate each generation's mutations during the simulation, or afterwards.
+Precisely, we need to know the genealogical trees relating all sampled individuals
 at each position along the genome.
-This is equivalent to knowing the \emph{population pedigree} --
-the complete history of parent-offspring relationships of an entire population
-going back to a remote time -- and the genetic outcomes of each ancestral
-meiosis.
-The embellished pedigree contains all the information necessary
-to construct the genealogical tree that relates each individual to every other
-at each position on the genome,
-i.e., the \emph{tree sequence} implemented in the coalescent simulator \msprime{}~\citep{kelleher2016efficient}.
 Combined with ancestral genotypes and the origins of new mutations,
-it also completely specifies the genomic sequence of any individual in the population at any time.
-This is much much more than typically recorded in a forward simulation, however,
-so further savings can be had by discarding all information irrelevant to the genetic history
+these trees completely specify the genomic sequence of any individual in the population at any time.
+To obtain this information, we record from forward simulation the \emph{population pedigree} --
+the complete history of parent-offspring relationships of an entire population
+going back to a remote time -- and the genetic outcomes of each ancestral meiosis,
+periodically discarding all information irrelevant to the genetic history
 of the extant population.
-\jda{The space complexity result seemed like a result in the intro; and maybe didn't include some detail
-	about assumed number of generations}
-
-In this paper, we describe storage methods for genealogies
-(and hence, genome sequence) and an algorithm for simplifying these,
-both implemented as improvements to the algorithmic tools and data structures in \msprime{}.
-We also describe a general method of using these tools to efficiently record,
-and later process, the population pedgiree in forwards-time simulation.
-While providing substantial savings in time and space, our methods provide much
-more information than simply simulating the genomes -- the tree sequence format
-encodes all marginal genealogies of individuals living at the end of the
-simulation.
+The information in this embellished pedigree is stored as a \emph{succinct tree sequence} 
+(or, for brevity, ``tree sequence''),
+which contains all the information necessary
+to construct the genealogical tree that relates each individual to every other
+at each position on the genome.
+
+In this paper, we describe a storage method for \emph{succinct tree sequences} 
+and, hence, genome sequence, as well as an algorithm for simplifying these.
+The data structure is \emph{succinct} in the sense that its' space usage is close to optimal,
```

Comment:
succinct here

Reply:
Perfect. I'll add a reference for succinct data strutures.